spec/README.md

```diff
@@ -186,6 +186,37 @@ Field Name | Type | Description
 <a name="parametersDefault"></a>default | `string` | **Optional** An optional default value for the parameter.
 <a name="parametersHidden"></a>hidden | `string` | **Optional** An optional boolean signifying the parameter should be obscured when displayed.
 
+##### XPathing
```

Comment:
Maybe we should remove xpathing all together and use pointers instead since most people would understand that better than xpathing.